### ISSUE TITLE
Question about JsonEscapeUtil.

### ISSUE THREAD

**Issue by zoooooway** [qos-ch/logback#861]
I noticed that `JsonEscapeUtil` has the following code:
```java
static String getObligatoryEscapeCode(char c) {
      if (c < 32)
          return ESCAPE_CODES[c];
      if (c == 0x22)
          return "\\\"";
      if (c == 0x5C)
          return "\\/";

      return null;
  }
```
I'm curious as to why the escape character for 0x5C is ”\\/” instead of "\\\\". Can anyone tell me why?

**Comment by ceki** [comment]
@zoooooway  It's a bug. Thank you for catching this. The return value should have been the java string "\\\\" or the string "\\" as you suggested.

**Comment by ceki** [comment]
Fixed in commit 5dfbb6185fdbc597ec0
